framework/doc/content/source/postprocessors/VectorPostprocessorDifferenceMeasure.md

```diff
@@ -0,0 +1,22 @@
+# VectorPostprocessorDifference
+
+!syntax description /Postprocessors/VectorPostprocessorDifference
+
+This post-processor is used to compare two vector post-processor vectors that
+have the same length. It returns a measure of the distance between the two
+vectors.
+There are two different options for the parameter `difference_type`. Denoting the first
+vector as `a` and the second as `b`, these options are as follows:
```

Comment:
I don't know how useful the "difference" norm will be without an absolute value operation -- the two vectors can be totally different and have zero "difference". I recommend adding Linfty norm as that's another commonly used measure for optimization. It would be even better if you could add a general vector p-norm.